### ISSUE TITLE
undefined method `include?' for nil:NilClass

### ISSUE THREAD

**Issue by CyborgMaster** [kickboxio/kickbox-ruby#9]
Over the weekend (the night of September 2nd) I got hundreds of these errors.  It seems likely to me that Kickbox had an availability issue and went down for a while, but I would expect these errors to be more descriptive an allow me to recover better than a nil error. 

Here's an example stack trace (email address redacted).

```
Sep 03 01:32:31 nova-background_i-60de5998 production.log: [2016-09-03T05:32:31.338866 #23985] ERROR Error verifing '********@hotmail.com' through Kickbox: undefined method `include?' for nil:NilClass
Sep 03 01:32:31 nova-background_i-60de5998 production.log: [2016-09-03T05:32:31.338950 #23985] ERROR /var/app/current/vendor/bundle/ruby/2.3.0/gems/kickbox-2.0.1/lib/kickbox/http_client/response_handler.rb:13:in `get_body'
Sep 03 01:32:31 nova-background_i-60de5998 production.log: /var/app/current/vendor/bundle/ruby/2.3.0/gems/kickbox-2.0.1/lib/kickbox/http_client/error_handler.rb:21:in `block in call'
Sep 03 01:32:31 nova-background_i-60de5998 production.log: /var/app/current/vendor/bundle/ruby/2.3.0/gems/faraday-0.9.2/lib/faraday/response.rb:57:in `on_complete'
Sep 03 01:32:31 nova-background_i-60de5998 production.log: /var/app/current/vendor/bundle/ruby/2.3.0/gems/kickbox-2.0.1/lib/kickbox/http_client/error_handler.rb:13:in `call'
Sep 03 01:32:31 nova-background_i-60de5998 production.log: /var/app/current/vendor/bundle/ruby/2.3.0/gems/kickbox-2.0.1/lib/kickbox/http_client/auth_handler.rb:34:in `call'
Sep 03 01:32:31 nova-background_i-60de5998 production.log: /var/app/current/vendor/bundle/ruby/2.3.0/gems/faraday-0.9.2/lib/faraday/connection.rb:377:in `run_request'
Sep 03 01:32:31 nova-background_i-60de5998 production.log: /var/app/current/vendor/bundle/ruby/2.3.0/gems/faraday-0.9.2/lib/faraday/connection.rb:140:in `get'
Sep 03 01:32:31 nova-background_i-60de5998 production.log: /var/app/current/vendor/bundle/ruby/2.3.0/gems/kickbox-2.0.1/lib/kickbox/http_client.rb:102:in `create_request'
Sep 03 01:32:31 nova-background_i-60de5998 production.log: /var/app/current/vendor/bundle/ruby/2.3.0/gems/kickbox-2.0.1/lib/kickbox/http_client.rb:101:in `instance_eval'
Sep 03 01:32:31 nova-background_i-60de5998 production.log: /var/app/current/vendor/bundle/ruby/2.3.0/gems/kickbox-2.0.1/lib/kickbox/http_client.rb:101:in `create_request'
Sep 03 01:32:31 nova-background_i-60de5998 production.log: /var/app/current/vendor/bundle/ruby/2.3.0/gems/kickbox-2.0.1/lib/kickbox/http_client.rb:85:in `request'
Sep 03 01:32:31 nova-background_i-60de5998 production.log: /var/app/current/vendor/bundle/ruby/2.3.0/gems/kickbox-2.0.1/lib/kickbox/http_client.rb:48:in `get'
Sep 03 01:32:31 nova-background_i-60de5998 production.log: /var/app/current/vendor/bundle/ruby/2.3.0/gems/kickbox-2.0.1/lib/kickbox/api/kickbox.rb:23:in `verify'
Sep 03 01:32:31 nova-background_i-60de5998 production.log: /var/app/current/app/lib/email_verification_service.rb:54:in `verify'
Sep 03 01:32:31 nova-background_i-60de5998 production.log: /var/app/current/app/lib/email_verification_service.rb:34:in `block (3 levels) in verify_batch'
Sep 03 01:32:31 nova-background_i-60de5998 production.log: /var/app/current/app/lib/email_verification_service.rb:33:in `each'
Sep 03 01:32:31 nova-background_i-60de5998 production.log: /var/app/current/app/lib/email_verification_service.rb:33:in `block (2 levels) in verify_batch'
```

**Comment by CyborgMaster** [comment]
I got another ~50 or so of these errors last night.  Any ideas?

**Comment by schaitanya** [comment]
@CyborgMaster FIxed this issue in kickbox-2.0.2. Please let us know if this worked for you .
